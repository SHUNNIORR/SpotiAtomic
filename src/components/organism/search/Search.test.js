import React from 'react';
import {shallow} from 'enzyme';
import Search from './Search';

describe('Pruebas de search',()=>{
    const wrapper = shallow(<Search></Search>);
    test('debe mostrarse correctamente',()=>{
        expect(wrapper).toMatchSnapshot();
    })
    test("El innput debe cambiar", ()=>{
        const input = wrapper.find('input');
        const value = 'Hola, soy un test';

        input.simulate('change', {target:{value}})
    })
})