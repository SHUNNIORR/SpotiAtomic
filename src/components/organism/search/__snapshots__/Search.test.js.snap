// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de search debe mostrarse correctamente 1`] = `ShallowWrapper {}`;
